jenkins/version-def.sh

```diff
@@ -26,6 +26,9 @@ for VAR in $OVERWRITE_PARAMS; do
 done
 IFS=$PRE_IFS
 
+# configs for Hybrid feature
```

Comment:
Better use default value patten instead of hardcode, then we can overwrite these vars outside without changing this script

```
GLUTEN_VERSION=${GLUTEN_VERSION:-"1.2.0"}
GLUTEN_FOR_OS=${GLUTEN_FOR_OS:-"20.04"}
```